### ISSUE TITLE
core dump error

### ISSUE THREAD

**Issue by xingwu2** [hanchenphd/GMMAT#11]
Hi, 

I am trying to run GMMAT(SMMAT) on a high-performance cluster. I followed the instruction and installed all the packages successfully. Then, I tried to repeat the examples from the manual, and I ran into issues. like this
<img width="839" alt="Screen Shot 2020-08-19 at 16 01 02" src="https://user-images.githubusercontent.com/14883511/90689281-6db5da80-e235-11ea-989c-3167de9ec60e.png">

Do you have any idea what causes the issue? Do I have to compile R with Intel MKL to run SMMAT? 

Best

Xing

**Comment by hanchenphd** [comment]
Hi Xing,

Thank you for your interest in GMMAT. You don't HAVE to compile R with MKL, but doing so will make your code run much faster.

I have very limited knowledge on core dump issues. It looks like a memory problem and you might need to work with your HPC admin to figure out what was going on. Which examples did you try? Could you please also try installing GMMAT on a different machine (server, desktop or laptop), and see if you can reproduce the issue?

Best,
Han

**Comment by xingwu2** [comment]
Hi Han,

It is now working on a desktop. However, I noticed an another issue here. Does the group file allow groups with only one marker?  Here is the error message. 

Error in group.idx.start[i]:group.idx.end[i] : argument of length 0

Best

Xing

**Comment by hanchenphd** [comment]
Hi Xing,

Yes, the program does allow groups with only one SNP. It was not the problem. This error usually implies chr:pos:ref:alt in your group definition file does not match your GDS file. Can you get chromosome, position and allele information from the GDS file and see if they match?

Best,
Han

**Comment by xingwu2** [comment]
Hi Han,

Thank you for you prompt reply.
I have another question. Due to the long computational time, can I use glmmkin function to build the model first, and then use SMMAT function to run each chromosome independently? Will the results from one chromosome affect the results from the other chromosomes? 

Best

Xing

**Comment by hanchenphd** [comment]
Hi Xing,

Yes, you can fit the null model using glmmkin( ) first, save the null model as an R object, and load it for each chromosome. Results from one chromosome do not depend on results from another chromosome, so you can run different chromosomes in parallel.

Best,
Han

**Comment by xingwu2** [comment]
Hi Han, 

Thank you so much for you response. 

Best

Xing
